sql-dump-parts Stuck on Empty XML file

When running `make sql-dump-parts` the script gets stuck on an empty xml file:
```
The errored article is  ( /home/wikipedia-mirror/drafts/errored_articles ). Fixing... (time: Sun Mar  8 22:53:29 EDT 2015)
ERROR: empty xml file /home/wikipedia-mirror/drafts/wikipedia-parts/enwiki-20140614-pages-meta-current1.xml-p000000010p000010000.fix.xml
XML parse script failed. This is serous. report this at
        http://github.com/fakedrake/wikipedia-mirror/issues
I didn't clean up after myself btw (rm /home/wikipedia-mirror/drafts/wikipedia-parts/enwiki-20140614-pages-meta-current1.xml-p000000010p000010000.fix.xml), will now exit 1
Finished 0th pass over /home/wikipedia-mirror/drafts/wikipedia-parts/enwiki-20140614-pages-meta-current1.xml-p000000010p000010000.fix.xml after removing 
I dont know what to do. I tried this 3 itmes, each time removing the bad article
Try re-running the command if this is java.lang.ArrayIndexOutOfBoundsException to continue
Otherwise please report at
        http://github.com/fakedrake/wikipedia-mirror/issues
The errored articles log is /home/wikipedia-mirror/drafts/errored_articles.
If it is empty or you see the same article more than once report your situation.
make: *** [/home/wikipedia-mirror/drafts/wikipedia-parts/enwiki-20140614-pages-meta-current1.xml-p000000010p000010000.sql] Error 1
```
Unfortunately, I don't know what article this is because both `/home/wikipedia-mirror/drafts/errored_articles` and the xml file itself (`/home/wikipedia-mirror/drafts/wikipedia-parts/enwiki-20140614-pages-meta-current1.xml-p000000010p000010000.fix.xml`) are empty files.

_What I've tried/looked at:_ 
I've already tried re-running `make sql-dump-parts` both with and without deleting the empty xml file from the disk, and I get the same error. Unless there's a better way around this, I would like to skip this xml file. 

In `Makefile.smartdumps` you mention
> Use ARTICLES_TO_REMOVE as a list of articles you know will fail to remove them before hand.

However, I don't see any way to use ARTICLES_TO_REMOVE, and the closest thing I could find is `xml-parse.sh`, yet I think it's `xml-parse.sh` itself that's complaining in the first place -- and further, I don't know the "TITLE_OF_ARTICLE_TO_REMOVE" (apparently, an argument needed for `xml-parse.sh`).

Any way to ignore this one XML file and continue the sql-dump-parts?

--- a/Makefile.smartdumps
+++ b/Makefile.smartdumps
@@ -31,34 +31,34 @@
 # enwiki-20131202-pages-articles20.xml-p011125004p013324998.bz2
 #WP_BASE_URL = http://dumps.wikimedia.org/enwiki/20131202
 
-WP_BASE_URL = http://dumps.wikimedia.org/enwiki/20140614
-WP_ALL_PAGES = enwiki-20140614-pages-meta-current1.xml-p000000010p000010000.bz2 \
-enwiki-20140614-pages-meta-current2.xml-p000010001p000025000.bz2 \
-enwiki-20140614-pages-meta-current3.xml-p000025001p000055000.bz2 \
-enwiki-20140614-pages-meta-current4.xml-p000055002p000104998.bz2 \
-enwiki-20140614-pages-meta-current5.xml-p000105001p000184999.bz2 \
-enwiki-20140614-pages-meta-current6.xml-p000185003p000305000.bz2 \
-enwiki-20140614-pages-meta-current7.xml-p000305002p000464997.bz2 \
-enwiki-20140614-pages-meta-current8.xml-p000465001p000665000.bz2 \
-enwiki-20140614-pages-meta-current9.xml-p000665001p000925000.bz2 \
-enwiki-20140614-pages-meta-current10.xml-p000925001p001325000.bz2 \
-enwiki-20140614-pages-meta-current11.xml-p001325001p001825000.bz2 \
-enwiki-20140614-pages-meta-current12.xml-p001825001p002425000.bz2 \
-enwiki-20140614-pages-meta-current13.xml-p002425001p003124998.bz2 \
-enwiki-20140614-pages-meta-current14.xml-p003125001p003924999.bz2 \
-enwiki-20140614-pages-meta-current15.xml-p003925001p004825000.bz2 \
-enwiki-20140614-pages-meta-current16.xml-p004825002p006025000.bz2 \
-enwiki-20140614-pages-meta-current17.xml-p006025001p007524997.bz2 \
-enwiki-20140614-pages-meta-current18.xml-p007525002p009225000.bz2 \
-enwiki-20140614-pages-meta-current19.xml-p009225001p011125000.bz2 \
-enwiki-20140614-pages-meta-current20.xml-p011125001p013324998.bz2 \
-enwiki-20140614-pages-meta-current21.xml-p013325001p015725000.bz2 \
-enwiki-20140614-pages-meta-current22.xml-p015725003p018225000.bz2 \
-enwiki-20140614-pages-meta-current23.xml-p018225001p020925000.bz2 \
-enwiki-20140614-pages-meta-current24.xml-p020925002p023725000.bz2 \
-enwiki-20140614-pages-meta-current25.xml-p023725001p026624999.bz2 \
-enwiki-20140614-pages-meta-current26.xml-p026625002p029625000.bz2 \
-enwiki-20140614-pages-meta-current27.xml-p029625001p043052726.bz2
+WP_BASE_URL = https://dumps.wikimedia.org/enwiki/20150205
+WP_ALL_PAGES = enwiki-20150205-pages-meta-current1.xml-p000000010p000010000.bz2 \
+enwiki-20150205-pages-meta-current2.xml-p000010001p000025000.bz2 \
+enwiki-20150205-pages-meta-current3.xml-p000025001p000055000.bz2 \
+enwiki-20150205-pages-meta-current4.xml-p000055002p000104998.bz2 \
+enwiki-20150205-pages-meta-current5.xml-p000105001p000184999.bz2 \
+enwiki-20150205-pages-meta-current6.xml-p000185003p000305000.bz2 \
+enwiki-20150205-pages-meta-current7.xml-p000305002p000464997.bz2 \
+enwiki-20150205-pages-meta-current8.xml-p000465001p000665000.bz2 \
+enwiki-20150205-pages-meta-current9.xml-p000665001p000925000.bz2 \
+enwiki-20150205-pages-meta-current10.xml-p000925001p001325000.bz2 \
+enwiki-20150205-pages-meta-current11.xml-p001325001p001825000.bz2 \
+enwiki-20150205-pages-meta-current12.xml-p001825001p002425000.bz2 \
+enwiki-20150205-pages-meta-current13.xml-p002425001p003124998.bz2 \
+enwiki-20150205-pages-meta-current14.xml-p003125001p003924999.bz2 \
+enwiki-20150205-pages-meta-current15.xml-p003925001p004825000.bz2 \
+enwiki-20150205-pages-meta-current16.xml-p004825002p006025000.bz2 \
+enwiki-20150205-pages-meta-current17.xml-p006025001p007524997.bz2 \
+enwiki-20150205-pages-meta-current18.xml-p007525002p009225000.bz2 \
+enwiki-20150205-pages-meta-current19.xml-p009225001p011125000.bz2 \
+enwiki-20150205-pages-meta-current20.xml-p011125001p013324998.bz2 \
+enwiki-20150205-pages-meta-current21.xml-p013325001p015725000.bz2 \
+enwiki-20150205-pages-meta-current22.xml-p015725003p018225000.bz2 \
+enwiki-20150205-pages-meta-current23.xml-p018225001p020925000.bz2 \
+enwiki-20150205-pages-meta-current24.xml-p020925002p023725000.bz2 \
+enwiki-20150205-pages-meta-current25.xml-p023725001p026624999.bz2 \
+enwiki-20150205-pages-meta-current26.xml-p026625002p029625000.bz2 \
+enwiki-20150205-pages-meta-current27.xml-p029625001p045319670.bz2
 
 # WP_BASE_URL = http://wikimedia.wansec.com/archive/enwiki/20080103/
 # WP_ALL_PAGES = enwiki-20080103-pages-articles.xml.bz2

